examples/anime-http/client/auth/trendingAnime.ts

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef */
+
+export async function clientAuth({ serverName }) {
+    console.log("serverName", serverName)
+  
+    return {
+      token: process.env.TOKEN,
+      oauth: process.env.OAUTH2,
```

Comment:
Currently are just able to send `OAuth` token from the client side right, is glee able to handle OAuth on server side?

Reply:
@Souvikns could you help me understand the question? We have the `params.getOauth()` method to retrieve the oauth token, Is there something else that's needed?